addons/kommander/1.163.x/kommander-2.yaml

```diff
@@ -0,0 +1,81 @@
+---
+apiVersion: kubeaddons.mesosphere.io/v1beta1
+kind: ClusterAddon
+metadata:
+  name: kommander
+  labels:
+    kubeaddons.mesosphere.io/name: kommander
+  annotations:
+    catalog.kubeaddons.mesosphere.io/addon-revision: "1.163.2-2"
```

Comment:
This annotation (and the file naming) is correct @natmegs ty! :+1:

Reply:
if we update version in file, folder name doesnt match anymore, can we rename folder to `1.x`?

also, there is #74 now 🙈